src/main/java/org/zendesk/client/v2/model/View.java

```diff
@@ -0,0 +1,128 @@
+package org.zendesk.client.v2.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class View implements Serializable {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
It's probably better to generate a random number. (IDEs are generally proposing it)

Reply:
fixed.